broker/fragment/stores.go

```diff
@@ -159,8 +159,15 @@ func parseStoreArgs(ep *url.URL, args interface{}) error {
 }
 
 func instrumentStoreOp(provider, op string, err error) {
+	// It appears that `errors.Cause` can sometimes return nil
```

Comment:
nit: switch this to use `errors.Unwrap` instead which is now part of the Go standard library since this was written.

Reply:
Oh neat, this is what I was missing. Thanks!